example/waf/bot_rules/bot_rules.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Edgecast Inc., Licensed under the terms of the Apache 2.0
+// license. See LICENSE file in project root for terms.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/EdgeCast/ec-sdk-go/edgecast"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/waf"
+)
+
+// Demonstrates the usage of WAF Bot Rules
+//
+// Usage:
+// go run bot_rules.go
+//
+// For detailed information about Bot Rules in WAF, please refer to:
+// TODO [URL TBD]
```

Comment:
Not sure if we should wait for the official documentation to be up or just leave this blank

Reply:
The TODO is okay if we create a followup ticket.

Reply:
URL has been updated